### PR TITLE
fix env check

### PR DESCRIPTION
**Pull request by MaximilianMeister**
Signed-off-by: Maximilian Meister <mmeister@suse.de>

### CODE REVIEW

File: jenkins-pipelines/Jenkinsfile.kubic-sandbox [hunk 1]
```diff
@@ -34,7 +34,7 @@ coreKubicProjectPeriodic(
         podName: 'default'
     )
 
-    if (env.RUN_CONFORMANCE_TESTS) {
+    if (RUN_CONFORMANCE_TESTS) {
```
Comment (kiall) [inline jenkins-pipelines/Jenkinsfile.kubic-sandbox]: Testing this out with replay, it looks like Jenkins treats all arguments as strings - the "booleanParam" just indicates which UI will be shown.

We'll need:

    if (RUN_CONFORMANCE_TESTS == "true") {

Comment (kiall) [reply jenkins-pipelines/Jenkinsfile.kubic-sandbox]: http://jenkins.caasp.suse.net/view/Misc%20Jobs/job/caasp-manual-sandbox/job/master/7 is rebuilding with this to verify:

    if (RUN_CONFORMANCE_TESTS == "true") {

Comment (kiall) [reply jenkins-pipelines/Jenkinsfile.kubic-sandbox]: `env.FOO` will work, but the variable starts in jenkins as just `FOO` and can be accessed either way... But `env.FOO` works - so merging :)

Comment (MaximilianMeister) [reply jenkins-pipelines/Jenkinsfile.kubic-sandbox]: i see, i was only wondering why http://jenkins.caasp.suse.net/view/Misc%20Jobs/job/caasp-manual-sandbox/job/master/7 didnt run the conformance tests... but i guess you've started it with default parameter (RUN_CONFORMANCE_TESTS=false)

**Pull request merged**